app/assets/javascripts/utilities/showModal.js

```diff
@@ -12,3 +12,10 @@ function showModal(context) {
   document.body.classList.add('modal-open');
   initSignupModal();
 }
+
+function showRateLimitModal() {
+    if(document.getElementsByClassName('crayons-modal')[0]) {
```

Comment:
Is it good practice for us to call the class name by its Crayons class name (which is descriptive of styles)? An alternative could be specifying a `rate-limit-modal` class and calling it that way.

@ludwiczakpawel @nickytonline

Reply:
[I'm a big fan of using `js-something` class names for purpose of calling an element in JS](https://dev-internal.slack.com/archives/C7GE84DEJ/p1579172524145300). That's also helpful with indicating whether or not an element has some JS associated.